crates/graphql/src/schema/query_root.rs

```diff
@@ -177,7 +176,14 @@ impl QueryRoot {
             .load(&conn)
             .context("Failed to load metadata")?;
 
-        Ok(rows.pop().map(Into::into))
+        Ok(rows.pop().map(|mut n| {
```

Comment:
Its cleaner do this swap in the image method of the nft object instead of in the dataloader.

Reply:
Added proposed changes in image method